test/verify/check-networkmanager-wireguard

```diff
@@ -30,7 +30,7 @@ class TestWireGuard(packagelib.PackageCase, netlib.NetworkCase):
         b.wait_visible("#network-wireguard-settings-dialog")
         iface_name = b.val("#network-wireguard-settings-interface-name-input")
         b.wait_visible("#network-wireguard-settings-save:disabled")
-        if m1.image in ["rhel4edge", "centos-8-stream"] or m1.image.startswith("rhel-8"):
+        if m1.image in ["rhel4edge"] or m1.image.startswith("rhel-8"):
```

Comment:
I've seen `ruff` complain about stuff like this before...

Reply:
OOI, what about?

Reply:
Ah! Can I say "let's use this as a reminder to kill it off completely"? :grin: At some point rhel4edge ought to catch up with RHEL 9.4 (it was just released in fact, so we should soon get a 9.4 based image).